### PR TITLE
EEC form content updates (EEC-3, EEC-8)

### PR DESCRIPTION
**Pull request by jamiecarterHO**
## What?

Added additional options to two fields in the form, see [EEC-3](https://collaboration.homeoffice.gov.uk/jira/browse/EEC-3) and [EEC-8](https://collaboration.homeoffice.gov.uk/jira/browse/EEC-8) for details of updates.

`'problem` field had options to raise an issue with the user's eVisa signin email or phone number added, and in the `requestor-reference` field a UKVI customer number option was added as the requestor's reference.

Support for these fields in validations and summary appearance was also added where necessary.

## Why?

Extra options were added to two fields in the form by the business after tickets were originally created, but the changes were  added in the original tickets. Raising this PR now before testing of the original work can take place (while we wait for DevOps pipeline to be built)

## Testing?

Can be tested under the original tickets

## Check list

- [x] I have reviewed my own pull request
- [ ] I have written tests (if relevant)

### CODE REVIEW

File: apps/eec/fields/index.js [hunk 1]
```diff
@@ -124,6 +135,26 @@ module.exports = {
...
+      'email',
+      { type: 'maxlength', arguments: 254 }
+    ],
+    className: ['govuk-input', 'govuk-!-width-two-thirds'],
+    dependent: {
+      field: 'problem',
+      value: 'problem-signin-email'
+    }
+  },
+  'detail-signin-phone': {
```
Comment (dk4g) [inline apps/eec/fields/index.js:150]: we can probably apply standard set of validators for phone number

```
if(phoneNumber.length > 0) {
        const phoneNumberWithoutSpace = phoneNumber.replace(/\s+/g, '').trim();
        const isValidphoneNumber = validators.regex(phoneNumberWithoutSpace, /^\(?\+?[\d()-]{8,16}$/);
        if(!isValidphoneNumber  || !validators.internationalPhoneNumber(phoneNumber)) {
          return validationErrorFunc('internationalPhoneNumber');
        }
      }
```

Comment (dk4g) [reply apps/eec/fields/index.js:150]: should this field be `required`?

File: apps/eec/fields/index.js [hunk 2]
```diff
@@ -124,6 +135,26 @@ module.exports = {
       value: 'problem-valid-until'
     }
   },
+  'detail-signin-email': {
+    mixin: 'input-text',
+    validate: [
```
Comment (dk4g) [inline apps/eec/fields/index.js:140]: should this field be `required`?

File: apps/eec/fields/index.js (hunk 1 above)
Comment (jamiecarterHO) [reply apps/eec/fields/index.js:150]: Currently they've asked for these fields not to be validated, and not to be required even if they are selected as options

File: apps/eec/fields/index.js (hunk 2 above)
Comment (jamiecarterHO) [reply apps/eec/fields/index.js:145]: Yes, UCD decided all these fields should be required if selected

File: apps/eec/fields/index.js (hunk 1 above)
Comment (jamiecarterHO) [reply apps/eec/fields/index.js:156]: Yes, all conditionally revealed fields in this page should now be required. `validations.json` updated as so.

Added HOF standard international/mobile validator

**Review by dk4g — APPROVED** [review]

**Pull request merged**